cprover_bindings/src/goto_program/typ.rs

```diff
@@ -1041,6 +1064,19 @@ impl Type {
         Pointer { typ: Box::new(self) }
     }
 
+    /// Convert type to its signed counterpart if possible.
+    /// For types that are already signed, this will return self.
```

Comment:
I don't think this is returning `Self` for `ssize_t`

Reply:
Right. I've added a case to return `Self` for that case, also in the symmetric method.